oxidation/client/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +14,8 @@
             android:name="com.scille.parsec.MainActivity"
             android:label="@string/title_activity_main"
             android:theme="@style/AppTheme.NoActionBarLaunch"
-            android:launchMode="singleTask">
+            android:launchMode="singleTask"
+            android:exported="false">
```

Comment:
What is this for ?

Reply:
It's to correct a deprecation error

```
/home/foobar/Documents/parsec-cloud/oxidation/client/android/app/src/main/AndroidManifest.xml:13:9-25:20 Error:
        android:exported needs to be explicitly specified for element <activity#com.scille.parsec.MainActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
```

tl,dr: the value `android:exported` need to be explicitly set